apps/gateway/src/routes/api/v1/projects/:projectId/commits/:commitUuid/documents/handlers/run.ts

```diff
@@ -54,6 +54,8 @@ export const runHandler = factory.createHandlers(
         },
       }).then((r) => r.unwrap())
 
+      await pipeToStream(stream, result.stream)
```

Comment:
Why this?

Reply:
Because the `createDocumentLog` job was being enqueued before the stream was even started. This caused it to not assign their `providerLogs` correctly, and to have wrong information such as duration.
Now we await the `pipeToStream` to finish, and then create the document log afterwards.

Reply:
Add a fix test

Reply:
No test 😢